lightkurve/search.py

```diff
@@ -46,13 +47,41 @@ def __init__(self, table=None):
             self.table = Table()
         else:
             self.table = table
-
-    def __repr__(self):
+            if len(table) > 0:
+                self._add_columns()
+
+    def _add_columns(self):
+        """Adds user-friendly ``idx`` and ``collection`` columns.
+        
+        These columns are not part of the MAST Portal API, but they make the
+        display of search results much nicer in Lightkurve."""
+        self.table['collection'] = None
+        self.table['#'] = None
+        try:
+            prefix = {'Kepler': 'Quarter', 'K2': 'Campaign', 'TESS': 'Sector'}
+            for idx in range(len(self.table)):
+                self.table['#'][idx] = idx
+                mission = self.table['obs_collection'][idx]
+                seqno = self.table['sequence_number'][idx]
+                if mission == 'Kepler' and self.table['sequence_number'].mask[3]:
+                    seqno = re.findall(r".*Q(\d+)", self.table['description'][idx])[0]
+                self.table['collection'][idx] = "{} {} {}".format(mission,
+                                                                  prefix[mission],
+                                                                  seqno)
+        except Exception as e:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'e' is assigned to but never used](https://app.codacy.com/app/barentsen/lightkurve/pullRequest?prid=4514383)